openerp-atomfeed-service/src/main/java/org/bahmni/feed/openerp/client/AtomFeedClientHelper.java

```diff
@@ -43,9 +49,10 @@ FeedClient getAtomFeedClient(Jobs jobName, FeedClientFactory feedClientFactory)
         AllFeeds allFeeds = getAllFeeds(atomFeedProperties, cookies);
         AllMarkers allMarkers = new AllMarkersJdbcImpl(transactionManager);
         AllFailedEvents allFailedEvents = new AllFailedEventsJdbcImpl(transactionManager);
-        //TODO: Create a toggle between rest and xml client
-        OpenERPContext openERPContext = new OpenERPContext(openERPXMLClient);
-        return feedClientFactory.getFeedClient(atomFeedProperties, transactionManager, openERPContext, allFeeds, allMarkers, allFailedEvents, jobName);
+        String isRestEnabled = environment.getProperty("IS_ODOO_16");
+        boolean isRestEnabledValue = Boolean.parseBoolean(isRestEnabled);
+        OpenERPContext openERPContext = isRestEnabledValue ? new OpenERPContext(openERPRESTClient) : new OpenERPContext(openERPXMLClient);
```

Comment:
Why is the variable called `openERPRestClient`? Maybe call it `odooRESTClient`?

Reply:
Makes sense will rename the class.